.github/scripts/_npm_publishlib.py

```
@@ -0,0 +1,511 @@
+                    f"{PACKAGE}@{version}",
+                    f"--registry={REGISTRY}",
+                ],
+                cwd=root,
+                text=True,
+                capture_output=True,
+                check=False,
+                timeout=60,
+            )
+            if installed.returncode != 0:
+                raise ValueError("npm signature verification install failed")
+            audited = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)